rudder-core/src/main/scala/com/normation/rudder/services/policies/RudderPromiseWriterServiceImpl.scala

```diff
@@ -156,13 +156,16 @@ class RudderCf3PromisesFileWriterServiceImpl(
    * From a base path and a base backup path, plus a node configuration, write the rules
```

Comment:
On that file, could you explain exactly what is done today, and if possible which file are doing that, so that the next time we know how to corrrect it:
- root server is special cases in class#method, new promises are written in /var/rudder/cfengine-community/inputs.new, then move to ... and old promises are moved to ....bkp
- normal nodes new promises etc etc

Wilth the quirks is any reminds. 

Thanks!